Lib/typing.py

```diff
@@ -428,8 +428,15 @@ def __getitem__(self, parameters):
         return self._getitem(self, *parameters)
 
 
-@_SpecialForm
-def Any(self, parameters):
+class _AnyMeta(type):
+    def __instancecheck__(self, obj):
```

Comment:
Should we even have this? `isinstance(X, Any)` is now a meaningful operation.

Reply:
I'm happy to remove it (as this PR currently does for `issubclass`). Doing so would also allow us to get rid of the metaclass, which will help remove restrictions on what classes can inherit from `Any`.

My reasoning for keeping it is that `isinstance` is very commonly used, potentially by typing / Python novices, and `isinstance(..., Any)` doesn't correspond well to the notion of `Any` at type check time. Sophisticated users have workarounds available to them.